source/features/open-issue-to-latest-comment.tsx

```diff
@@ -12,13 +13,15 @@ function init(): void {
 }
 
 function initDashboard(): void {
-	for (const icon of select.all('.js-recent-activity-container :not(a) > div > .octicon-comment')) {
-		const url = icon.closest('li')!.querySelector('a')!.pathname + '#partial-timeline';
-		const link = <a className="muted-link" href={url}/>;
-		icon.parentElement!.classList.remove('col-1'); // Also fix extra space added by GitHub #3174
-		icon.parentElement!.append(link);
-		link.append(icon, icon.nextSibling!);
-	}
+	observe('.js-recent-activity-container :not(a) > div > .octicon-comment', {
+		add(icon) {
+			const url = icon.closest('li')!.querySelector('a')!.pathname + '#partial-timeline';
+			const link = <a className="muted-link" href={url}/>;
+			icon.parentElement!.classList.remove('col-1'); // Also fix extra space added by GitHub #3174
+			icon.parentElement!.append(link);
+			link.append(icon, icon.nextSibling!);
```

Comment:
I think this could be also replaced with
```suggestion
			wrapAll(link [icon, icon.nextSibling!);
```